.github/workflows/lint.yml

```diff
@@ -0,0 +1,24 @@
+name: Linting on PR
+on:
+  pull_request:
+    branches:
+      - '**'
+    
+jobs:
+
+  Continuous-Integration:
+
+    name: Performs linting on the application
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the Repository
+        uses: actions/checkout@v3
```

Comment:
Looks like we need to use actions/checkout@v4 (at least). Node.js version bump.

Reply:
done @Relequestual